client/my-sites/upgrades/checkout/checkout.jsx

```diff
@@ -306,12 +320,22 @@ const Checkout = React.createClass( {
 	}
 } );
 
+function mapStateToProps( state ) {
+	const cards = getStoredCards( state );
+	const selectedSite = getSelectedSite( state );
+	const selectedSiteId = getSelectedSiteId( state );
+	const selectedSiteSlug = getSelectedSiteSlug( state );
```

Comment:
Shall I inline each selector here? I could also inline the whole mapStateToProps function as in the [upgrade-nudge](https://github.com/Automattic/wp-calypso/pull/9332/commits/3b80e179ca2e02c84bfee509f0404a895a3e50bc#diff-5360467136ddd69da0add2ed9ff1e5adR143) PR.